09week/towers-of-hanoi/index.html

```diff
@@ -6,18 +6,19 @@
   <link rel="stylesheet" href="style.css" />
```

Comment:
For a checkpoint I was expecting a little bit more into styling and adding a little more to the application.